### ISSUE TITLE
new domain creation error

### ISSUE THREAD

**Issue by alemumol** [inverse-inc/packetfence#8244]
after I create a new domain, it pops up below error while testing.

Post "http://containers-gateway.internal:5000/ntlm/connect": context deadline exceeded (Client.Timeout exceeded while awaiting headers).

I can also telnet this service  'containers-gateway.internal:5000'.

![2](https://github.com/user-attachments/assets/a1d66f25-fb28-4aa3-9138-1010e0c3b79e)

**Comment by stgmsa** [comment]
Hi @alemumol,
could you provide the details when hitting this error ?
PacketFence version,
Fresh install or an upgrade 
What is the linux version that has PacketFence installed on

This seems to be a network issue, and the information above can help us reproduce the error. 
Thanks!

**Comment by alemumol** [comment]
thanks for your reply stgmsa,
PacketFence version= ZEN-v13.3.0
With Debian version = 11 and it is a fresh install

**Comment by JeGoi** [comment]
@alemumol, you mean ZEN-v13.2.0 on debian 11, correct ?
There is no v13.3 and Packetfence next iteration, the v14.0, is on debian12.

**Comment by stgmsa** [comment]
I think it's debian 14.0 on debian 11. 
@lzammit  and I met this issue on v14 dev + debian, its' due to a chroot issue. but since v14 will be released based on debian12, there won't be an issue like this.

**Comment by JeGoi** [comment]
So, I need to remove debian11 on 14.0.
Thanks

**Comment by wtatanka** [comment]
I also have the same problem on a new installation made like the iso of version 14.0 do you have the solution
(Post "http://containers-gateway.internal:5000/ntlm/connect": context deadline exceeded (Client.Timeout exceeded while awaiting headers))

**Comment by JeGoi** [comment]
@wtatanka you mean, you used the packetfence iso 14.0 and have the same issue, correct?
So, you are under debian12, correct?

**Comment by wtatanka** [comment]
Yes debian 12
root@ALPANAC01:~# lsb_release -a
No LSB modules are available.
Distributor ID: Debian
Description:    Debian GNU/Linux 12 (bookworm)
Release:        12
Codename:       bookworm
root@ALPANAC01:~# 

Paolo Traballoni 

> Il giorno 2 ott 2024, alle ore 19:47, JeGoi ***@***.***> ha scritto:
> 
> R

**Comment by wtatanka** [comment]
you can sugest solution?
root@ALPANAC01:~# telnet containers-gateway.internal 5000
Trying 100.64.0.1...
Connected to containers-gateway.internal.
Escape character is '^]'.
get
<!DOCTYPE HTML>
<html lang="en">
    <head>
        <meta charset="utf-8">
        <title>Error response</title>
    </head>
    <body>
        <h1>Error response</h1>
        <p>Error code: 400</p>
        <p>Message: Bad request syntax ('get').</p>
        <p>Error code explanation: 400 - Bad request syntax or unsupported method.</p>
    </body>
</html>
Connection closed by foreign host.
root@ALPANAC01:~# 


I solved the problem

**Comment by MarioSpenc** [comment]
Same problem here: Install a fresh 14.0 from your ISO, Import data from running version 13.1, but when connecting to AD i get error:

![image](https://github.com/user-attachments/assets/3ff4d50e-1a1c-4487-bf7a-2e261f1797d7)

**Comment by JeGoi** [comment]
@MarioSpenc ISO has been updated on "2024-10-07 20:56". Are you using the last ISO?
Please, check md5sum to be sure.
I will prepare VMs to test that part.
Thanks for feedbacks.

**Comment by MarioSpenc** [comment]
Hi, md5sum is good/the same. 

What the workaround is: we have set the machine account password manually - in AC with powershell, in PF in domain.conf. That works, machine is joined. Something went wrong when joining in our AD ...

**Comment by stgmsa** [comment]
@MarioSpenc 
Thanks for the feedback, we'll verify both the cases mentioned.

**Comment by wtatanka** [comment]
Commet # all line on resolve.confPaolo Traballoni Il giorno 10 ott 2024, alle ore 14:20, stgmsa ***@***.***> ha scritto:﻿
@MarioSpenc
Thanks for the feedback, we'll verify both the cases mentioned.

**Comment by JeGoi** [comment]
Hum, I am not able to reproduce.
I test it with new fresh install iso 14.0 on esx, (-/+ export/import) script + weird and insane password...
Always working...

**Comment by MarioSpenc** [comment]
JeGoi, I think the problem could be a subbug of https://github.com/inverse-inc/packetfence/issues/8301#issuecomment-2347071482 . There is something changing in Hashmanagement in new sssd. 

When I change the machine account password in Active Directory and the machine account password in domain.conf, it works.

/Mario

**Comment by JeGoi** [comment]
@MarioSpenc Could you try to change the fingerprint of the machine?
Perhaps related to: https://github.com/inverse-inc/packetfence/issues/8339
But should be on zen not iso...
```
rm /etc/ssh/ssh_host_*key*
ssh-keygen -A
```
And not sure it is related... just a try

**Comment by satkunas** [comment]
We're unable to reproduce, more information is needed.

**Comment by brandonechenique** [comment]
dear community, 
I make the connection with the domain ad and at the beginning I get a test ok domain joined, but after a few seconds I get the following error:
Post “http://containers-gateway.internal:5000/ntlm/connect”: context deadline exceeded (Client.Timeout exceeded while awaiting headers)

already tried the following :
restarting the service ntlm-auth-api
changing machine password

I've installed the latest version of packetfence 14.0 and the debian version is 12.

in the server ad, i can see that the packetfence object was created in the computers group
from the server pf i make  telnet to port 389 to the ad server and it connects.

What else could I check? or what steps should I do to generate a correct tshoot?

**Comment by brandonechenique** [comment]
in addition to that, I cannot generate a source authenticcation based on the ad, I always get an error with the following information:
Can't connect to server or bind with '“CN=NACPF_RADIUS2,OU=RADIUS,OU=Groups,DC=xxxxx,DC=xxxx,DC=xxxx,DC=xx”' on x.x.x.x.x.:389

**Comment by MarioSpenc** [comment]
Hi!

My recommendation is NOT to USE version 14.0 in production, due to several bugs with Active Directory integration, in special with AD structures with more than one AD server. It's nearly impossible to get it work stable. 

/Mario

**Comment by brandonechenique** [comment]
@MarioSpenc  thanks for your recomendation

**Comment by brandonechenique** [comment]
I am now testing version 11.2, the synchronization with the ad is stable, and I have already configured the roles and authentication rules.
I added a sw, and when I try to test the access from a windows machine I get an error, the pf logs are as follows
[radius-acct.log](https://github.com/user-attachments/files/17777679/radius-acct.log)
![image](https://github.com/user-attachments/assets/c4b9d1dd-3ec2-41ce-8930-68083d8026b8)

what could be the problem?

**Comment by JeGoi** [comment]
@brandonechenique For support, please use the https://www.packetfence.org/support.html#/community or https://www.packetfence.org/support.html#/commercial

That been wrote: it is written in your log file: "Error binding to port for 192.168.35.28 port 1813"
Try that:
```
lsof -iTCP:1813
lsof -iUDP:1813
```

Thank you for using PacketFence

**Comment by stgmsa** [comment]
failures due to dc node selection, temp disabled and fixed by 7ee63c3c
